readme.md

```diff
@@ -623,86 +689,46 @@ interface RequestCredential {
             /** optional list of DIDs of attesters trusted by this verifier */
             trustedAttesters?: string[]
 
-            /** list of credential attributes which MUST be included when submitting the credential */
+            /**
+             * list of credential attributes which MUST be included when submitting the credential.
+             * The properties are identified using JSONPath.
+             */
             requiredProperties: string[]
         }
     ]
 
     /** Optional DID URI the credential should be issued to */
-    owner?: string
+    subject?: string
 
     /** 24 random bytes as hexadecimal */
     challenge: string
 }
-
-const exampleRequest: RequestCredential = {
-    "cTypes": [
-        {
-            "cTypeHash": "0x5366521b1cf4497cfe5f17663a7387a87bb8f2c4295d7c40f3140e7ee6afc41b",
-            "trustedAttesters": [
-                "did:kilt:5CqJa4Ct7oMeMESzehTiN9fwYdGLd7tqeirRMpGDh2XxYYyx"
-            ],
-            "requiredProperties": [
-                "name"
-            ]
-        }
-    ],
-    "challenge": "9f1ceac971cce4c61505974f411a9db432949531abe10dde"
-}
 ```
 
+##### Required Properties
+
+The `cTypes.requiredProperties` property specifies which fields of the credential MUST be included in the presentation.
+All fields which are not covered by the selective disclosure scheme are mandatory and MUST not be listed in the `requiredProperties`.
+
+
 #### 2. Extension or dApp sends credential
 
 The extension MUST only send the credential with active consent of the user.
 This is the first step where the user’s DID is revealed to the dApp.
 
-The `challenge` from the previous message MUST be used to update the `claimerSignature`
+The `challenge` from the previous message MUST be used to create the [verifiable presentation](https://www.w3.org/TR/vc-data-model/#presentations-0)
```

Comment:
we could add that this would be added to the data integrity proof on its `challenge` field